docs/FAQ.md

```diff
@@ -102,6 +102,14 @@ hub pull someone_else
 
 It is not always straightforward to tell `gem` to look in non-standard directories for headers and libraries. If you choose `/usr/local`, many things will "just work".
 
+## Why does Homebrew on Linux recommend that I install to `/home/linuxbrew/.linuxbrew`?
+
+Homebrew's pre-built binary packages, called bottles, of most software can only be used if you install in `/home/linuxbrew/.linuxbrew`, otherwise they have to be built from source. Building from source takes a long time, is prone to fail, and is not supported. Install Homebrew on Linux in `/home/linuxbrew/.linuxbrew/` if possible so that you can use our pre-built binary packages. If you do not yourself have admin privileges, consider asking your admin staff to create a `linuxbrew` role account for you with home directory `/home/linuxbrew`.
```

Comment:
```suggestion
Homebrew's pre-built binary packages (known as bottles) of most software can only be used if you install in `/home/linuxbrew/.linuxbrew`, otherwise they have to be built from source. Building from source takes a long time, is prone to fail, and is not supported. Install Homebrew on Linux in `/home/linuxbrew/.linuxbrew/` if possible so that you can use our pre-built binary packages. If you do not yourself have admin privileges, consider asking your admin staff to create a `linuxbrew` role account for you with home directory `/home/linuxbrew`.
```

I think this is my final edit. 😆

Reply:
Done!